src/NIST-800-53/rules/redshift/nist80053RedshiftClusterConfiguration.ts

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import { CfnCluster } from '@aws-cdk/aws-redshift';
+import { CfnResource, Stack } from '@aws-cdk/core';
+
+/**
+ * Redshift clusters have encryption and audit logging enabled
+ * @param node the CfnResource to check
+ */
+export default function (node: CfnResource): boolean {
+  if (node instanceof CfnCluster) {
+    const encrypted = Stack.of(node).resolve(node.encrypted);
+    const loggingProperties = Stack.of(node).resolve(node.loggingProperties);
+    if (!encrypted || encrypted == undefined || loggingProperties == undefined) {
```

Comment:
`!undefined` evaluates so we can shorten this
```suggestion
    if (!encrypted  || loggingProperties == undefined) {
```